src/constants/networks.ts

```diff
@@ -63,10 +65,21 @@ export const PolygonNetworkInfo: NetworkInfo = {
   imageURL: POLYGON_LOGO_URL,
   blurb: '',
 }
+export const CeloNetworkInfo: NetworkInfo = {
+  id: SupportedNetwork.CELO,
+  route: 'celo',
+  name: 'Celo',
+  bgColor: '#2E3338',
```

Comment:
nit: Just want to verify this, as it looks very plain.

Reply:
One of our design folks is revisiting this